kaspresso/src/main/kotlin/com/kaspersky/kaspresso/interceptors/behaviorkautomator/impl/flakysafety/FlakySafeObjectBehaviorInterceptor.kt

```diff
@@ -30,7 +30,10 @@ class FlakySafeObjectBehaviorInterceptor(
         interaction: UiObjectInteraction,
         assertion: UiObjectAssertion,
         activity: () -> T
-    ): T = flakySafely(action = activity)
+    ): T = flakySafely(action = {
+        interaction.reFindUiObject()
```

Comment:
Improvements of Kautomator stability

Reply:
1. This one must be relocated to a separate Interceptor. Thanks that you've highlighted.
2. Good question about stability. So, the concept is similar to the one that we had in Loader...Interceptor. But, the architecture solution is better and the same with Jetpack Compose Intercepting (see old weird methods like `loadView` and etc.). Better stability is achieved due to changing the order of behavior interceptors (try to reload earlier). I think, there were cases when SystemDialog and Scrolling Interceptors consumed all allocated time, that's why there was no time to reload Element. But, I don't have tests where I can check it.